internal/config/config.go

```diff
@@ -207,35 +220,49 @@ func (o *Override) Parse() error {
 			}
 		}
 		if !found {
-			return fmt.Errorf("Package override `go_type` specifier %q is not a Go basic type e.g. 'string'", o.GoType)
+			return fmt.Errorf("Package override `go_type` specifier %q is not a Go basic type e.g. 'string'", o.GoTypeParam.TypeName)
 		}
 		o.GoBasicType = true
 	} else {
-		// assume the type lives in a Go package
-		if lastDot == -1 {
-			return fmt.Errorf("Package override `go_type` specifier %q is not the proper format, expected 'package.type', e.g. 'github.com/segmentio/ksuid.KSUID'", o.GoType)
-		}
-		if lastSlash == -1 {
-			return fmt.Errorf("Package override `go_type` specifier %q is not the proper format, expected 'package.type', e.g. 'github.com/segmentio/ksuid.KSUID'", o.GoType)
-		}
-		typename = o.GoType[lastSlash+1:]
-		if strings.HasPrefix(typename, "go-") {
+		// TODO need to implement checks for #177, is an import path is set, the package is external otherwise it doesn't need to be imported
+		/*		This checks shouldn't be needed anymore, proper Go type definition responsibility is to the end user given the explicitly set fields.
```

Comment:
This comment isn't correct since we need to continue to support a world where `go_type` is a string.